ramls/hridsettings.json

```
@@ -24,6 +24,11 @@
       "description": "The HRID settings for items",
       "type": "object",
       "$ref": "hridsetting.json"
+    },
+    "commonRetainLeadingZeroes" : {
```

I think the location of this property already makes this property common across all record types and that makes the `common` prefix unnecessary.

Please feel free to ignore this if you don't agree with the feedback or want the explicit `common` part in the name.

```suggestion
    "retainLeadingZeroes" : {
```